tools/gce-change-disktype/README.md

```diff
@@ -0,0 +1,45 @@
+# Update GCE Instance disk-type
+
+Update disks attached to a GCE instance.
```

Comment:
Script to change disk type of disks attached to a GCE instance.

Reply:
Fixed.